src/main/java/lotto/model/LottoNumber.java

```diff
@@ -0,0 +1,38 @@
+package lotto.model;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    public static final int LOWER_BOUND = 1;
+    public static final int UPPER_BOUND = 45;
+    private final int number;
```

Comment:
상수와 인스턴스 변수는 줄바꿈을 통해서 구별하기 쉽게해주세요. 😉

```suggestion
    public static final int UPPER_BOUND = 45;
    
    private final int number;
```

Reply:
반영하였습니다!